source/includes/api/tables/progress-response.rst

```diff
@@ -9,7 +9,8 @@
 
    * - ``state``
      - string
-     - The current state of {+c2c-product-name+}.
+     - The current state of {+c2c-product-name+}. For information on the
```

Comment:
I'm not sure where {c2c-product-name} is, but can we use "mongosync" throughout this page?